tests/test_preprocessing/test_interval_encoder.py

```diff
@@ -15,10 +15,6 @@
             general_checks,
             estimator_checks.check_estimators_dtypes,
             estimator_checks.check_fit_score_takes_y,
-            # estimator_checks.check_dtype_object,
-            # estimator_checks.check_sample_weights_pandas_series,
-            # estimator_checks.check_sample_weights_list,
-            # estimator_checks.check_sample_weights_invariance,
```

Comment:
Please don't remove these. By leaving these comments here, we're conscious of the sklearn tests that we've manually removed.

Reply:
Ok, no problem. Added them back in. 

I also noticed some print statements in `test_patsy_transformer.py`. Are they also supposed to be there?